docs/production/mobile-push-notifications.md

```diff
@@ -1,38 +1,44 @@
+arrived. This is an important feature for having a great mobile app experience.
 
-For technical reasons (explained below), in order to deliver mobile
-push notifications in the app store versions of our mobile apps, you
-will need to register your Zulip server with the Zulip mobile push
-notification service. This service will forward push notifications
-generated by your server to the Zulip mobile app automatically.
+To set up mobile push notifications, you will need to register your Zulip server
+with the Zulip mobile push notification service. This service will forward push
+notifications generated by your server to users' mobile apps.
 
 ## How to sign up
 
-Starting with Zulip 1.6 for both Android and iOS, Zulip servers
-support forwarding push notifications to a central push notification
-forwarding service. Accessing this service requires outgoing HTTPS
-access to the public Internet; if that is restricted by a proxy, you
-will need to [configure Zulip to use your outgoing HTTP
-proxy](deployment.md#customizing-the-outgoing-http-proxy)
-first.
+You can enable the mobile push notification service for your Zulip server as
+follows:
 
-You can enable this for your Zulip server as follows:
+1. Make sure your server has outgoing HTTPS access to the public Internet. If
+   that is restricted by a proxy, you will need to [configure Zulip to use your
+   outgoing HTTP proxy](deployment.md#customizing-the-outgoing-http-proxy)
+   first.
+
+1. Decide whether to upload basic usage statistics. Systems using the Mobile
+  Push Notifications Service will, by default, submit basic usage statistics
+  (e.g. Zulip version, number of users, number of messages sent) to the service.
+  These statistics help Zulip's maintainers understand how many people are
+  self-hosting Zulip in order to allocate resources towards supporting
+  self-hosted installations.
+
+    Our use of these statistics is governed by the same [Terms of
+    Service](https://zulip.com/policies/terms) and [Privacy
+    Policy](https://zulip.com/policies/privacy) that covers the Mobile Push
+    Notifications Service itself. If your organization does not want to submit these
+    statistics, you can disable this feature during setup or at any time by setting
+    `SUBMIT_USAGE_STATISTICS=False` in `/etc/zulip/settings.py`.
 
 1. Uncomment the
    `PUSH_NOTIFICATION_BOUNCER_URL = 'https://push.zulipchat.com'` line
-   in your `/etc/zulip/settings.py` file (i.e. remove the `#` at the
+   in your `/etc/zulip/settings.py` file (i.e., remove the `#` at the
    start of the line), and [restart your Zulip
-   server](settings.md#making-changes). If you
-   installed your Zulip server with a version older than 1.6, you'll
-   need to add the line (it won't be there to uncomment).
+   server](settings.md#making-changes).
 
-1. If you're running Zulip 1.8.1 or newer, you can run the
-   registration command:
+1. Run the registration command:
 
    ```bash
    # As root:
```

Comment:
I don't think it's useful to show two different ways of doing the same thing, followed by a third way which is specific to another deployment strategy.  I might say:
````
1. Run the registration command.  If you installed Zulip directly on the server (without Docker), run as root:
   ```
   su zulip -c '/home/zulip/deployments/current/manage.py register_server'
   ```
   Or if you're using Docker, run:
   ```
   docker exec -it -u zulip <container_name> /home/zulip/deployments/current/manage.py register_server
   ```
````

Reply:
Done.